pybloqs/block/base.py

```diff
@@ -186,7 +185,7 @@ def save(
 
         if is_html:
             content = self.render_html(static_output=False, header_block=header_block, footer_block=footer_block)
-            with open(filename, "w", encoding="utf-8") as f:
+            with builtins.open(filename, "w", encoding="utf-8") as f:
```

Comment:
I think this can just be `open` without having to import `builtins`

Reply:
😲 I thought `UP` would have caught that as it did for some other files. Will remedy.